Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.cpp

```diff
@@ -424,16 +426,23 @@ namespace AzToolsFramework
                                     patchEntry, patchesMetadata.m_instancesToAdd, patchesMetadata.m_instancesToRemove, AZStd::move(patchPath));
                             }
                         }
+                        // ContainerEntity
                         else if (patchPath.starts_with(PathMatchingContainerEntity)) // Path begins with /ContainerEntity
                         {
                             patchesMetadata.m_shouldReloadContainerEntity = true;
                         }
+                        // LinkId
+                        else if (patchPath == PathMatchingLinkId) // Path is /LinkId
+                        {
+                            continue;
```

Comment:
Suggestion: I'm not sure having a continue here makes the most sense since the for loop could want to do more after this if/else block in the future. Another option would be to do "else if (patchPath != PathMatchingLinkId) { show error }"

Reply:
Good suggestion, but imo would be good to keep `patchPath == PathMatchingLinkId` to have consistency with other comparisons.

I took a closer look in the for loop. imo it is still better to just "continue" which matches our intent for link id at this time.

But I will update the for loop to this:

```cpp
for (const PrefabDomValue& patchEntry : patches.GetArray())
{
    PrefabDomValue::ConstMemberIterator patchEntryIterator = patchEntry.FindMember("path");
    if (patchEntryIterator == patchEntry.MemberEnd())
    {
        continue;
    }
    
    // Path Comparison
}
```